src/app/(sidebar)/transaction/fee-bump/page.tsx

```diff
@@ -1,5 +1,200 @@
+        }
+      }
+    }
+  };
+
+  return (
+    <Box gap="md">
+      <div className="PageHeader">
+        <Text size="md" as="h1" weight="medium">
+          Fee Bump
+        </Text>
+
+        <Button
+          size="md"
+          variant="error"
+          icon={<Icon.RefreshCw01 />}
+          iconPosition="right"
+          onClick={() => {
+            resetBaseFee();
+          }}
+        >
+          Clear and import new
+        </Button>
+      </div>
+      <Card>
+        <Box gap="lg">
+          <PubKeyPicker
+            id="source_account"
+            label="Source Account"
+            value={source_account}
+            error={sourceErrorMessage}
+            onChange={(e) => {
+              const error = validate.publicKey(e.target.value);
+              setSourceErrorMessage(error || "");
+              updateBaseFeeSource(e.target.value);
+            }}
+            note={<>The account responsible for paying the transaction fee.</>}
+            infoLink="https://developers.stellar.org/docs/learn/glossary#source-account"
+          />
+
+          <PositiveIntPicker
+            id="fee"
+            label="Base Fee"
+            value={fee}
+            error={baseErrorMessage}
+            onChange={(e) => {
+              const error = validate.positiveInt(e.target.value);
+              setBaseErrorMessage(error || "");
+              updateBaseFeeBase(e.target.value);
+            }}
+            note={
+              <>
+                The{" "}
+                <SdsLink href="https://developers.stellar.org/docs/learn/fundamentals/fees-resource-limits-metering">
+                  network base fee
+                </SdsLink>{" "}
+                fee is currently set to 100 stroops (0.00001 lumens). Based on
+                current network activity, we suggest setting it to 100 stroops.
+                Final transaction fee is equal to base fee times number of
+                operations in this transaction.
+              </>
+            }
+            infoLink="https://developers.stellar.org/docs/learn/glossary#base-fee"
+          />
+
+          <XdrPicker
+            id="submit-tx-xdr"
+            label="Input a base-64 encoded TransactionEnvelope:"
+            value={innerTxXdr}
+            error={xdrError}
+            onChange={(e) => onXdrChange(e.target.value)}
+            note="Enter a base-64 encoded XDR blob to decode."
+            hasCopyButton
+          />
+        </Box>
+      </Card>
+      <>
+        {feeBumpedTx.xdr ? (
+          <ValidationResponseCard
+            variant="success"
+            title="Success! Transaction Envelope XDR:"
+            response={
+              <Box gap="xs">
+                <TxResponse
+                  label="Network Passphrase:"
+                  value={network.passphrase}
+                />
+                <TxResponse label="XDR:" value={feeBumpedTx.xdr} />
+              </Box>
+            }
+            note={<></>}
```

Comment:
Could we make `note` optional in `ValidationResponseCard`?

Reply:
it is optional but i added it to get the padding for the buttons